File: loadercomplex-forge/ver12/src/main/java/com/enderzombi102/loadercomplex/forge12/impl/utils/ForgeServer.java
package com.enderzombi102.loadercomplex.forge12.impl.utils;

import com.enderzombi102.loadercomplex.api.minecraft.util.Server;
import net.minecraft.server.MinecraftServer;

public class ForgeServer implements Server {
	private final MinecraftServer backingServer;

	public ForgeServer(MinecraftServer minecraftServer) {
		backingServer = minecraftServer;
	}

	@Override
	public Object getObject() {
		return backingServer;
	}
}

File: loadercomplex-forge/src/ver182/kotlin/com/enderzombi102/loadercomplex/forge182/impl/server/ForgeServer.kt
package com.enderzombi102.loadercomplex.forge182.impl.server

import com.enderzombi102.loadercomplex.api.minecraft.server.Server
import net.minecraft.server.MinecraftServer

class ForgeServer(private val backingServer: MinecraftServer) : Server {
	override fun getObject(): Any {
		return this.backingServer
	}
}
